test/Microsoft.AspNet.Mvc.IntegrationTests/ByteArrayModelBinderIntegrationTest.cs

```diff
@@ -92,8 +92,9 @@ private class Person
                 ParameterType = typeof(byte[])
             };
 
-            // No data is passed.
-            var operationContext = ModelBindingTestHelper.GetOperationBindingContext(httpContext => { });
+            // No data is passed. The ByteArrayModelBinder should ensure no later binders run.
+            var operationContext = ModelBindingTestHelper.GetOperationBindingContext(
+                updateOptions: ModelBindingTestHelper.UpdateOptionsToEnsureNothingFollows<ByteArrayModelBinder>);
             var modelState = new ModelStateDictionary();
```

Comment:
I don't see this method as part of the PR. Also, the purpose of these tests is to test what model binding does with the default/typical configuration. If we need to do esoteric things to these tests it's probably just better suited for a unit test.